imgix/drift
# Fix bug #220: disable handleTouch doesn't work

## Description

Disabling handleTouch is not working, if you set to false, the default value true will always be used, and the same goes for injectBaseStyles.

- Fix issue #220   

## Checklist

Please use the checklist that is most closely related to your PR _(you only need to use one checklist, and you can skip items that aren't applicable or don't make sense)_:

### Bug Fix

 All existing unit tests are still passing (if applicable)

## Steps to Test

Just set handleTouch to false and check the result, a more appropriate way is to edit the test case "accepts custom settings", like below to include "handleTouch" and "injectBaseStyles", and then run tests:
```js
it("accepts custom settings", () => {
      const anchor = document.querySelector(".test-anchor");
      const drift = new Drift(anchor, { inlineOffsetX: 12, handleTouch: false,
                                        injectBaseStyles: false });

      const expectedConfig = defaultDriftConfig();
      expectedConfig.inlineOffsetX = 12;
      expectedConfig.handleTouch = false;
      expectedConfig.injectBaseStyles = false;

      expect(drift.settings).toEqual(expectedConfig);
    });
```

Related issue: #220
